test/games_collection_test.rb

```diff
@@ -106,4 +110,43 @@ def test_it_can_return_hash_of_average_goals_per_season
                     }
     assert_equal expected_hash, @games_collection.average_goals_per_season
   end
+
+  def test_it_can_tell_us_each_unique_home_team_id
+    #this validates that we're getting a condensed list of teams
+    assert_equal 17, @games_collection.home_teams.length
+  end
+
+  def test_it_can_tell_us_number_of_goals_per_home_team
+    assert_instance_of Hash, @games_collection.home_team_goals
```

Comment:
We chatted about this in person, but for the benefit of the team: yeah, I'd beef this test up a bit. Maybe also test the length of the hash, the data types of the keys and values, and other stuff you can test without writing out the whole hash.